lsp-sample/client/src/extension.ts

```diff
@@ -0,0 +1,62 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.

```

Comment:
Is MIT license compatible with creative commons or whatever SASyLF is currently distributed under?